content/spaces/avatar.textile

```diff
@@ -15,9 +15,10 @@ h2(#events). Event types
 The following four event types are emitted by members:
 
 - @enter@ := A new member has entered the space. The member has either entered explicitly by calling "@space.enter()@":/spaces/space#enter, or has attempted to update their profile data before entering a space, which will instead emit an @enter@ event.
-- @update@ := A member has updated their profile data by calling "@space.updateProfileData()@":/spaces/space#update-profile.
+- @updateProfile@ := A member has updated their profile data by calling "@space.updateProfileData()@":/spaces/space#update-profile.
 - @leave@ := A member has left the space. The member has either left explicitly by calling "@space.leave()@":/spaces/space#leave, or has abruptly disconnected and not re-established a connection within 15 seconds.
 - @remove@ := A member has been removed from the members list after the "@offlineTimeout@":/spaces/space#options period has elapsed. This enables members to appear greyed out in the avatar stack to indicate that they recently left for the period of time between their @leave@ and @remove@ events.
+- @update@ := This event is emitted whenever any one of the above events is emitted.
```

Comment:
I'm not sure I understand the use case for `update` now. Is this not just duplicating the use of `members.subscribe()` without subscribing to specific event types?

Reply:
yes, that's correct. I'd have to dig out the conversation from slack, but the old behaviour was confusing for developers

Reply:
Added 67f4bac014c1de01511d9ff52cd84454f1b7f71c to address this everywhere if you could take a look @dpiatek